pkgdown/_pkgdown.yml

```diff
@@ -14,6 +14,7 @@ authors:
 
 template:
   bootstrap: 4
+  trailingslash_redirect: true
```

Comment:
to be removed.

Reply:
BTW I find it helpful to add a check list to the PR so I don't forget to undo these sorts of changes.